cmd/gin-repod/repo.go

```diff
@@ -837,7 +827,13 @@ func (s *Server) putRepoCollaborator(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// jquery 1.9+ ajax calls require a proper JSON response body, otherwise they will default to error.
+	returnBody, _ := json.Marshal(struct {
+		Response string `json:"Response"`
+	}{"Success"})
+
```

Comment:
Seems like overkill instead of doing a  `io.WriteString(w, "{\"Response\": \"Success\"}")` at line 836.